microsoft/testsuites/dpdk/dpdktestpmd.py

```diff
@@ -838,19 +839,22 @@ def _install_ninja_and_meson(self) -> None:
                 "Failed to update Meson to latest version with pip3"
             ),
         )
-        if node.shell.exists(node.get_pure_path("/usr/bin/meson")):
-            node.tools[Mv].move(
-                "/usr/bin/meson", "/usr/bin/meson.bak", overwrite=True, sudo=True
+
+        if node.shell.exists(node.get_pure_path("/usr/local/bin/meson")):
+            if node.shell.exists(node.get_pure_path("/usr/bin/meson")):
```

Comment:
Add comments to explain why move it can copy the local one here. If it doesn't need to recover, how about deleting it directly? If the test case runs multiple times, the file will be overwritten anyway.

Reply:
added